torch/_inductor/fx_passes/pre_grad.py

```diff
@@ -59,6 +60,9 @@ def pre_grad_passes(gm, example_inputs):
     # used to implement low memory dropout
     gm = overrides.replace_fx(gm, example_inputs)
 
+    # Preprocessing for quantization on CPU. Skipped if gm is not quantized.
+    gm = quantization_pre_grad_pass(gm, example_inputs)
```

Comment:
Can we do this as a post-grad pass?

Pre-grad IR's are dangerous to work with because they have not been functionalized.  Thus many passes are unsound and will be buggy with aliasing/mutation.

Reply:
Thanks. If we want to do it in post-grad pass, const-folding is needed. However, the const-folding functionality is still under development (https://github.com/pytorch/pytorch/pull/100652). So, we remove this for now.